src/Goerli-DssSpellCollateral.sol

```diff
@@ -28,61 +61,257 @@ contract DssSpellCollateralAction {
-        //         ilk:                   ,
-        //         gem:                   ,
-        //         join:                  ,
-        //         clip:                  ,
-        //         calc:                  ,
-        //         pip:                   ,
-        //         isLiquidatable:        ,
-        //         isOSM:                 ,
-        //         whitelistOSM:          ,
-        //         ilkDebtCeiling:        ,
-        //         minVaultAmount:        ,
-        //         maxLiquidationAmount:  ,
-        //         liquidationPenalty:    ,
-        //         ilkStabilityFee:       ,
-        //         startingPriceFactor:   ,
-        //         breakerTolerance:      ,
-        //         auctionDuration:       ,
-        //         permittedDrop:         ,
-        //         liquidationRatio:      ,
-        //         kprFlatReward:         ,
-        //         kprPctReward:
-        //     })
-        // );
-
-        // DssExecLib.setStairstepExponentialDecrease(
-        //     CALC_ADDR,
-        //     DURATION,
-        //     PCT_BPS
-        // );
-
-        // DssExecLib.setIlkAutoLineParameters(
-        //     ILK,
-        //     AMOUNT,
-        //     GAP,
-        //     TTL
-        // );
-
-        // ChainLog Updates
-        // Add the new flip and join to the Chainlog
-        // address constant CHAINLOG        = DssExecLib.LOG();
-        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
-        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
-        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+    uint256 constant ZERO_PCT_RATE           = 1000000000000000000000000000;
+    uint256 constant ZERO_ZERO_FIVE_PCT_RATE = 1000000000015850933588756013;
+
+    // --- Math ---
+    uint256 public constant WAD = 10**18;
+    uint256 public constant RAY = 10**27;
+    uint256 public constant RAD = 10**45;
+
+    // -- RWA008 MIP21 components --
+    address constant RWA008                    = 0x30434AA15F85598F45406d08A79dCdD9a79335e9;
+    address constant MCD_JOIN_RWA008_A         = 0x4ce65E856f824C2b4a2eeD88E79b839eB366967d;
+    address constant RWA008_A_URN              = 0x6351915f840937Edba75656727f20165185FaB83;
+    address constant RWA008_A_URN_CLOSE_HELPER = 0xb220461eFFa9c0b1D09047D08Bf116DfbD9814fA;
+    address constant RWA008_A_INPUT_CONDUIT    = 0x6298a2498b4E3789bb8220Cf5f5b44f518509e2a;
+    address constant RWA008_A_OUTPUT_CONDUIT   = 0x1FE95E519F0CE8eCF1cdC885f4DeA7913e146149;
+    // SocGen's wallet
+    address constant RWA008_A_OPERATOR         = 0x3F761335890721752476d4F210A7ad9BEf66fb45;
+    // DIIS Group wallet
+    address constant RWA008_A_MATE             = 0xb9444802F0831A3EB9f90E24EFe5FfA20138d684;
+
+    string  constant RWA008_DOC                = "QmdfzY6p5EpkYMN8wcomF2a1GsJbhkPiRQVRYSPfS4NZtB";
+    uint256 constant RWA008_A_INITIAL_PRICE    = 30_437_069 * WAD;
```

Comment:
I can't fine the formula we use to compute this.  IIRC, it's two years of the rate applied to the DC, but we had a `bc` formula to get it as a comment.  Let's put that comment in so it's easy to validate.

Reply:
Your comment made me go double check and this value is wrong. It was calculated using `0.5%` SF while the actual value is `0.05%`. I'm gonna re-calculate and amend the value. I'll make sure to include the formula in a comment.

Reply:
Done.